src/main/java/controller/Game.java

```diff
@@ -0,0 +1,40 @@
+package controller;
+
+import domain.Cars;
+import domain.WinnerSelector;
+import view.View;
+
+public class Game {
+
+    public void startGame() {
+        try {
+            String inputNames = View.inputCarName();
+            int tryCount = View.inputTryCount();
+
+            Cars cars = new Cars();
+
+            //입력 받은 자동차 이름 분리 및 자동차 생성
+            cars.splitCarName(inputNames);
+            cars.generateCar();
+
+            //게임 시작 및 게임 과정 출력
+            Game(cars, tryCount);
+
+            //우승자 출력
+            WinnerSelector winnerSelector = new WinnerSelector();
+            View.printWinner(winnerSelector.getWinners(cars.getCars()));
+
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+        }
+
+
+    }
+    //생성자를 통해 게임 시작
+    public void Game(Cars cars, int tryCount){
```

Comment:
소문자 camelCase 동사형으로 작성해주세요!
```suggestion
    public void playGame(Cars cars, int tryCount){
```